pkg/fanal/secret/scanner_test.go

```diff
@@ -462,8 +462,8 @@ func TestSecretScanner(t *testing.T) {
 			Lines: []types.Line{
 				{
 					Number:      1,
-					Content:     "{\"key\": \"-----BEGIN RSA PRIVATE KEY-----**************************************************************************************************************************-----END RSA PRIVATE KEY-----\\n\"}",
-					Highlighted: "{\"key\": \"-----BEGIN RSA PRIVATE KEY-----**************************************************************************************************************************-----END RSA PRIVATE KEY-----\\n\"}",
+					Content:     "----BEGIN RSA PRIVATE KEY-----**************************************************************************************************************************-----END RSA PRIVATE",
+					Highlighted: "----BEGIN RSA PRIVATE KEY-----**************************************************************************************************************************-----END RSA PRIVATE",
```

Comment:
@DmitriyLewen I've decreased `maxLineLength` and updated the test. 
it shows before/after.